samples/grpc-server-aot/gradlew

```diff
@@ -0,0 +1,252 @@
+#!/bin/sh
```

Comment:
I would create a sample w/ "gradle" + "grpc" + "native" in start.spring.io and use whatever ".gradle + wrapper" pieces it generates (I believe ours is a bit outdated currently).